File: Hivemind/Contracts/IncomeReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hivemind.Contracts
{
    public class IncomeReport
    {
        public IEnumerable<TerritoryIncomeReport> Gross { get; set; }
        public int GiantKillerBonus { get; set; }
        public int Income { get; set; }
    }
}

File: Hivemind.Ng/src/autogenerated/entities/IncomeReport.ts
import { TerritoryIncomeReport } from './TerritoryIncomeReport';

export class IncomeReport {
    public gross: TerritoryIncomeReport[];
    public giantKillerBonus: number;
    public income: number;

    public constructor(partial: Partial<IncomeReport>) {
        if (partial.gross) {
            this.gross = partial.gross;
        }
        if (partial.giantKillerBonus) {
            this.giantKillerBonus = partial.giantKillerBonus;
        }
        if (partial.income) {
            this.income = partial.income;
        }
    }
}